src/AzureDevOpsDemoBuilder/appsettings.Development.json

```diff
@@ -1,51 +1,9 @@
 {
```

Comment:
😨 Was this configuration in the repository?!
Have we cycled the secrets?

Reply:
Yes these were the old keys used initially and recycled long ago. Now we are not maintaining app setting in repo. I'll remove these file from Cache.

Reply:
👍